### PR TITLE
Add Focus Appearance and Alt Text outcomes first draft

### PR DESCRIPTION
**Pull request by iadawn**
Content taken from [Focus Appearance (Exploratory to Developing)](https://docs.google.com/document/d/1H5Fk9QRy6WutIUyrCRcqXH04mkp2EFq_qeRXvMawWf0/edit?usp=sharing) and [Text Alternatives (Exploratory to Developing)](https://docs.google.com/document/d/1wkQ8ZKiRnNnax9ENaDXfQhl6kh-iYlXx7gMEhKMTQBU/edit#heading=h.cu8l0ohk9n2q)

### CODE REVIEW

**Comment by PoornimaSubramanian** [comment]
Hi! Couple of things to check in here - 

**For Focus appearance:**
1. One suggestion for the below TBC line in Focus Appearance to add 'in the viewport' or 'of viewport' whichever makes it easier to understand - 
            Not obscured: The focus indicator is not obscured or partially obscured (**more than 50% in the viewport,** TBC)

2. A small typo to correct the spelling of 'indicators' in the below line - 
            5. Is the indicator using lines or a bounding box? (Baseline)
Yes, see [line / bounding box **indiators**]

3. Question: Is this referring to whether the indicator showing (or) using the tooltip?
            Does the indicator **use a tooltip** or additional content as the indicator? (Baseline)

**For Text alternatives:**
1. Question: Should there be a method to call out 'images that contain text in it'?

**Comment by bruce-usab** [comment]
I feel you are heading in right direction.  In particular, it is helpful to now have outcome Focus Appearance as a more realized model Keyboard Input to follow.

**Comment by rachaelbradley** [comment]
We need to rework this to better reflect the realignment of guidelines, outcomes and methods.

**Comment by julierawe** [comment]
Hi, folks, thanks so much for all of the great work on this so far! I huddled with a few members of COGA, and we have some flags about the outcome template/formatting:

### **Hierarchy of information**

**Outcome name** (“Focus appearance”) is hard to notice at the top of the page.

- It’s a smaller font/seems less important than “Get started.”
- It’s far away from all of the information about this outcome.

**“Get started”** sounds like a second step (“Take action/What to do”) instead of first step (“Purpose/Why to do it).”

- Consider changing the name of “Get started” to “Why” or “Overview” or “Introduction.” 

**Box just below “Get started”** needs more context to connect it to the rest of page.

- Suggest adding “Outcome,” which is currently not mentioned on this page other than in the breadcrumbs. 
- The new wording would be: “Outcome: The keyboard focus must have a sufficient visual indicator.”

**“Methods”** page would benefit from a tweak that makes two things clear:

- The “Decision tree” is a series of questions that will help users determine which method to use. 
- There is a “Methods” section below the decision tree.
- “See More” surprised us as link text—instead of a CTA button to another page, could this be a collapsible section?

**“User needs”** seems like it should come before “Methods”

### **One page vs many tabs**
The current template has a lot of tabs.

- This adds complexity in general and users can lose orientation.
- This also makes it harder to do things like print out the guidance for this outcome. 
- Possible to put all of the “Focus appearance” information on one page with collapsible sections?

### **Colors**
**Red**

- The red bullets and numbering are very concerning to COGA.
- Scrolling with red can be harmful for users—it has the potential for accidental stroking or movement problems.
- The red is also very problematic when viewed in dark mode. 
- Red can also have different meanings in different cultures.

**Light blue/gray**

- The box around the one-sentence outcome description is very hard to see.
- This box is also hard to see in dark mode, as medium blue on black.
- It’s also hard to see the light blue/gray outline around each method on the "Methods" tab.

### **Breadcrumb formatting**
Right now there is no visual emphasis to help users what page they’re on.

- Consider boldfacing the part of the breadcrumb the user is currently on to help the user knows where they are.
- Consider adding an icon that identifies what the "home page" is (to the left of "WCAG 3 Outcomes").

**Comment by ChrisLoiselle** [comment]
Question on Prerequisites being presented prior to baseline or enhanced and use of overall use of these words. A numbered list shows that an enhanced option precedes a prerequisite whereas the define term is either a thing that is required as a prior condition for something else to happen or exist. or required as a prior condition. So if using a numbered list, why would these be all intermixed in a numbering system within the focus appearance?

**Review by scottaohara — COMMENTED** [review]
just some comments for my first pass through. 

some comments aren't scoped to the content of the drafts, but some of the markup being used to signify the pages are drafts.

probably need to do a few more reads of this to make sure i don't have other thoughts.

File: how-to/_includes/layout/outcome-nav.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="sidenav">
+  <nav class="sn-contents" aria-label="Outcome navigation">
```
[the remark this answers is not in the document]
Comment (iadawn) [reply how-to/_includes/layout/outcome-nav.html]: Fixed with [dd1009b](https://github.com/w3c/wcag3/pull/112/commits/dd1009bd3bdccf6207fcc968bd88ffddd8af0bd0)

File: how-to/_includes/layout/header.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+</section>
+<div class="minimal-header-container default-grid">
+  
+  <header class="minimal-header" id="site-header">
+    <div class="minimal-header-name">
+      <a href="/">WCAG 3 How-Tos</a>
+    </div>
+    <div class="minimal-header-logo">
+      <a href="http://w3.org/" aria-label="W3C">
+        <svg xmlns="http://www.w3.org/2000/svg" height="2em" viewBox="0 0 91.968 44">
```
[the remark this answers is not in the document]
Comment (iadawn) [reply how-to/_includes/layout/header.html]: Fixed with 55bbd3b

File: how-to/outcomes/focus-appearance/methods.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+1. Is there a focus indicator?
+    1. Yes, continue
+    2. No, fail
+
+2. Does the organization have a style guide which includes guidance on all types of text-alternatives, and a policy that editors are required to follow the style guide?
+    1. Yes, See Text Alternative Style Guide Assertion  (Enhanced), continue
+    2. No, continue
+
+3. Is the user-agent default focus indicator used? (Prerequisite)
```
[the remark this answers is not in the document]
Comment (iadawn) [reply how-to/outcomes/focus-appearance/methods.md]: Removed in a73aeed

@rachaelbradley Might be something for a style guide?

File: how-to/_includes/layout/header.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
+<a href="#main" class="button button--skip-link">Skip to content</a>
+<section class="default-grid info" tabindex="0" lang="en">
```
[the remark this answers is not in the document]
Comment (iadawn) [reply how-to/_includes/layout/header.html:2]: What 'this' are you referring to?

[the remark this answers is not in the document]
Comment (scottaohara) [reply how-to/_includes/layout/header.html:2]: the line i'm commenting on.  the section element.

[the remark this answers is not in the document]
Comment (iadawn) [reply how-to/_includes/layout/header.html]: Fixed with 1c6d7b2

**Review by alastc — APPROVED** [review]

**Pull request merged**